File: LEETCODE/1480. 一维数组的动态和.java
class Solution {
    public int[] runningSum(int[] nums) {
        int[] res = new int[nums.length];
        res[0] = nums[0];
        for(int i=1;i<res.length;i++) {
            res[i] = res[i-1] + nums[i];
        }
        return res;
    }
}

File: LEETCODE/kotlin/1480. 一维数组的动态和.kt
class Solution {
    fun runningSum(nums: IntArray): IntArray {
        var res = IntArray(nums.size)
        res[0] = nums[0]
        for(i in 1..res.size-1)
            res[i] = res[i-1] + nums[i]
        return res
    }
}